src/pages/home/sign-up/index.js

```diff
@@ -40,7 +40,11 @@ export default function SignUp() {
         e.preventDefault();
         setIsLoading(true);
         const request = signUp(requestBody);
-        request.then(() => history.push("/"));
+        request.then((res) => {
+            const user = JSON.stringify(res.data);
+            localStorage.setItem('user', user);
+            history.push("/timeline");
+        });
```

Comment:
Aqui, acho que não precisa jogar a info do sign-up para o Local Storage, já que já está jogando quando o usuário faz o signin (o que tá lá é sobrescrito pelo sign-in).

Reply:
Aqui eu quis errar pelo excesso mesmo, já que o sign-up retorna o token, dá pra de repente jogar o usuário direto pra timeline após o cadastro e manter a conexão, se não também não faz diferença